server/webSocketServer/index.js

```diff
@@ -0,0 +1,33 @@
+const WebSocket = require('ws');
+const userHolder = require('./userHolder');
+const messageCreator = require('./messageCreator');
+
+const startWebSocketServer = (httpServer) => {
+
+    const webSocketServer = new WebSocket.Server({ server: httpServer });
+    const users = userHolder();
```

Comment:
usersHolder? maybe userscontroller..

Reply:
but it only holds users and controller could be confusing, espaciely if im gonna have a rest server there also.
if not holder do you prefer? Container, storage?

Reply:
container is better, yes